### PR TITLE
Dev/issue 8894 premis parsing

### PR DESCRIPTION
**Pull request by mcantelon**
Added PREMIS object/event parsers.

### CODE REVIEW

**Comment by Hwesta** [comment]
Note: Previous iterations of this feature found in #12 #13 and #14

File: metsrw/premis/event.py [hunk 1]
```diff
@@ -0,0 +1,262 @@
...
+
+        return identifiers
+
+    def serialize(self):
+        root = self._document_root()
+        root.append(self._serialize_identifier())
+
+        event_type_el = etree.Element(utils.lxmlns('premis') + 'eventType')
+        event_type_el.text = self.event_type
+        root.append(event_type_el)
```
Comment (Hwesta) [inline metsrw/premis/event.py]: Tip: These three lines can be `etree.SubElement(root, utils.lxmlns('premis') + 'eventType').text = self.event_type`

**Comment by jrwdunham** [comment]
This should be closed in favour of https://github.com/artefactual-labs/mets-reader-writer/pull/34, which adds these changes to current master and makes them compatible with the recently-introduced plugin/ dependency injection pattern. Once the comments/ CR from this PR are transferred to PR 34, this can be closed.

**Comment by jrwdunham** [comment]
Closed in favour of https://github.com/artefactual-labs/mets-reader-writer/pull/34

**Pull request closed without merging**